pyproject.toml

```diff
@@ -0,0 +1,101 @@
+[build-system]
+requires = ["setuptools", "torch == 1.12", "numpy", "pybind11"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "torchopt"
+description = "A Jax-style optimizer for PyTorch."
```

Comment:
lack long description from readme, it's a must

Reply:
The long description is specified by `readme = "README.md"` below.